salary/src/main/resources/application.properties

```diff
@@ -1,11 +1,28 @@
+quarkus.http.port = 8082
+
+# configure your datasource
 quarkus.datasource.db-kind = postgresql
-quarkus.datasource.username = sarah
-quarkus.datasource.password = connor
-quarkus.datasource.jdbc.url = jdbc:postgresql://localhost:5432/mydatabase
+quarkus.datasource.username = postgres
+quarkus.datasource.password = pass
+quarkus.datasource.jdbc.url = jdbc:postgresql://localhost:5432/salary
 
 # drop and create the database at startup (use `update` to only update the schema)
 quarkus.hibernate-orm.database.generation = update
 
+# core service
+core-service/mp-rest/url=http://localhost:8080/
+core-service/mp-rest/scope=javax.inject.Singleton
```

Comment:
Scopes can be declared directly in Code with annotations? If you would like for it to be Singleton please replace annotation instead.

Reply:
Yes I know. XStefans used it like this in example project. I can rewrite it, if you want.

Reply:
If you could, since this is a static variable I would much rather see it defined in code